cdap-docs/developers-manual/source/building-blocks/datasets/index.rst

```diff
@@ -138,7 +141,23 @@ Tables. These Tables are similar to tables in a relational database with a few k
   and written independently of other columns, and columns are ordered
   in byte-lexicographic order. They are also known as *Ordered Columnar Tables*.
 
-|fileset|_ is a core Dataset type that was added as an experimental feature in CDAP 2.6.0.
+
+A FileSet represents a collections of files in the file system that share some common attributes
+such as the format and schema, while abstracting from the actual underlying file system interfaces.
+
+- Every file in a FileSet is in a location relative to the FileSet's base directory.
+
+- Knowing a file's relative path, any program can obtain a ``Location`` for that file through a method
+  of the FileSet dataset. It can then directly interact with the file's Location, for example write
```

Comment:
It can then interact directly with the file's Location; for example, to write data to the Location, or to read data from it.